src/main/java/org/broadinstitute/hellbender/utils/SimpleInterval.java

```diff
@@ -36,6 +36,60 @@ public SimpleInterval(final String contig, final int start, final int end){
         this.end = end;
     }
 
+    /**
+     * Makes an interval by parting the string. The required format is
+     * contig or contig:start or contig:start-end
+     *
+     * example:
+     *
+     * 'chr2', 'chr2:1000000' or 'chr2:1,000,000-2,000,000'
+     */
+    public SimpleInterval(final String string){
+        Locatable loc = parse(string);
```

Comment:
After discussion, we agreed that the best solution here is to inline the `parse()` code in the constructor, since `parse()` is already creating a new interval object. This would also allow us to keep the instance variables final.